tests/stress_tests/scripts/requirements.txt

```diff
@@ -2,6 +2,7 @@ pymongo>=3.12.0
 Jinja2>=2.11.2
 PyYAML>=5.4.1
 fastjsonschema~=2.15.1
-pandas>=1.1.5
+pandas>=1.3.5
 h5py>=3.1.0
-scipy~=1.5.4
\ No newline at end of file
+scipy~=1.7; python_version == '3.7'
```

Comment:
is it possible to align with solution what used for other components?
scipy >=1.5.4,<=1.8.0

Reply:
There were a lot of issues with the wrong scipy version being installed in CI pipelines. We had to specify a different version for Python 3.7, that's the only way we were able to satisfy pip dependency resolver.